### PR TITLE
basic plasma shapes now plotted on left

### PR DESCRIPTION
**Pull request by billingsley-john**
PR which plots basic plasmas on the left and plasmaboundaries plasmas on the right

### CODE REVIEW

**Comment by billingsley-john** [comment]
As described in #174, more plasmas can be added when values for major radius and minor radius are provided

**Comment by billingsley-john** [comment]
Single and double null NSTX plasmas have been added to the make_plasmas script using these input [parameters](https://github.com/RemiTheWarrior/plasma-boundaries/blob/master/plasmaboundaries/parameters.py). However, single null and double null NSTX plasmas look very similar (single null doesn't look like a single null plasma)

**Comment by RemiTheWarrior** [comment]
@billingsley-john could you provide some screenshots ? do you obtain something like this ? 
image

**Comment by RemiTheWarrior** [comment]
> Single and double null NSTX plasmas have been added to the make_plasmas script using these input [parameters](https://github.com/RemiTheWarrior/plasma-boundaries/blob/master/plasmaboundaries/parameters.py). However, single null and double null NSTX plasmas look very similar (single null doesn't look like a single null plasma)

@billingsley-john the default configuration of the `make_plasma_plasmaboundaries` function is `'single-null`. Change this parameter in order to get the correct configuration

File: examples/example_parametric_components/make_plasmas.py [hunk 1]
```diff
@@ -202,12 +202,34 @@ def main():
...
     )
 
+    NSTX_double_null_plasma_plasmaboundaries = make_plasma_plasmaboundaries(
+        name="NSTX_double_null_plasma_plasmaboundaries",
+        A=0,
+        major_radius=850,
+        minor_radius=680,
+        triangularity=0.35,
+        elongation=2,
+        color="yellow"
```
Comment (RemiTheWarrior) [inline examples/example_parametric_components/make_plasmas.py:216]: in order to obtain a double null configuration, add config='double-null'

**Comment by billingsley-john** [comment]
> > Single and double null NSTX plasmas have been added to the make_plasmas script using these input [parameters](https://github.com/RemiTheWarrior/plasma-boundaries/blob/master/plasmaboundaries/parameters.py). However, single null and double null NSTX plasmas look very similar (single null doesn't look like a single null plasma)
> 
> @billingsley-john the default configuration of the `make_plasma_plasmaboundaries` function is `'single-null`. Change this parameter in order to get the correct configuration

Ah, I see! Thank you very much, making this change seems to work. Thank you very much @RemiTheWarrior

**Pull request merged**